core/Test/Tasty/Ingredients/ConsoleReporter.hs

```diff
@@ -192,7 +192,8 @@ buildTestOutput opts tree =
           when getAnsiTricks $ do
             putChar '\r'
             clearLine
-            putStr testNamePadded
+            withoutLineWrap $
+              putStr testNamePadded
```

Comment:
What's the behavior of switching line wrapping off and on before finishing the line? We are going to print `resultShortDescription` later on.

Reply:
I'll investigate this as well as guarding the line wrap changes with CPP.

Reply:
I think we want `resultShortDescription` to also be printed without line wrap, so I'll put that in here.

Reply:
I've verified that in at least my terminal, if I disable line wrap, then print enough text to ordinarily wrap, then enable line wrap and print some more text, the additional text after line wrap is enabled is wrapped and nothing else. I think that's what we want.